### PR TITLE
docs(checklist): add pr review checklist

### PR DESCRIPTION
**Pull request by BirgitBader**
Relates to [ottoapi_docs otto-ec/ottoapi_orga#91](https://github.com/otto-ec/ottoapi_orga/issues/91)

Adds checklist to review a pull request in the api-guidelines repository.

### CODE REVIEW

**Comment by DominiquePetrich1709** [comment]
Maybe something like:
- Is the change in line with our API guidelines?

**Comment by BirgitBader** [comment]
> Maybe something like:
> 
> * Is the change in line with our API guidelines?

@DominiquePetrich1709 Thanks! Is this a suggestion for this API Guidelines checklist or for an ottoapi_docs review?

**Comment by DominiquePetrich1709** [comment]
> > Maybe something like:
> > 
> > * Is the change in line with our API guidelines?
> 
> @DominiquePetrich1709 Thanks! Is this a suggestion for this API Guidelines checklist or for an ottoapi_docs review?

@BirgitBader It's for the pr review checklist.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,6 +89,14 @@ Changelog:
 
 ```
 
+## Review a pull request (checklist)
+
+- Does the added/changed information belong where it is placed?
+- Does the change introduce security vulnerabilities, sensitive data or internal information?
```
Comment (cgebken) [inline CONTRIBUTING.md:95]:
```suggestion
- Does the change introduce security vulnerabilities, sensitive data, or internal information?
```

**Pull request merged**